src/server/common/dbconn.cpp

```diff
@@ -109,6 +110,64 @@ std::string dbconn::get_tournaments()
 	}
 }
 
+std::unique_ptr<simple_wml::document> dbconn::get_game_history(int player_id, int offset)
+{
+	try
+	{
+		std::string game_history_query = "select "
+"  game.GAME_NAME, "
+"  game.RELOAD, "
+"  game.START_TIME, "
+"  GROUP_CONCAT(CONCAT(player.USER_NAME, ':', player.FACTION)) as PLAYERS, "
+"  IFNULL(scenario.NAME, '') as SCENARIO_NAME, "
+"  IFNULL(scenario.ID, '') as SCENARIO_ID, "
+"  IFNULL(era.NAME, '') as ERA_NAME, "
+"  IFNULL(era.ID, '') as ERA_ID, "
+"  IFNULL(GROUP_CONCAT(distinct mods.NAME), IFNULL(GROUP_CONCAT(distinct mods.ID), '')) as MODIFICATIONS, "
```

Comment:
Perhaps this should also be split in name and ID so the client can decide how to display this?

Reply:
Updated.